layouts/partials/head.html

```diff
@@ -3,6 +3,23 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
 
+    <!-- GrowthBook / Segment keys -->
+    <script>
+        {{- if eq hugo.Environment "production" }}
+            var segmentWriteKey = "UK90Ofwacetj5VCPJ7cUgkbNcKLSHO3u";
+            var snippetName = "C8Y429BDEy/06ujkUhzfL.min.js";
+            window.growthbook_sdk_key="sdk-psPofGr6jFV2ja9O";
```

Comment:
Keys in source code??

Reply:
I know it looks wonky, but it's common practice since these are frontend keys (Used by the frontend library) so they are always available to end users anyway; you can see this as a common practice with any library like Segment, Google Analytics, Mixpanel, etc.